softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorization/impl/cache/AuthorizationCacheService.java

```diff
@@ -0,0 +1,17 @@
+package com.softgroup.authorization.impl.cache;
+
+import com.softgroup.common.cache.impl.CacheService;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 20.03.17.
+ */
+@Service
+public class AuthorizationCacheService extends CacheService<RegistrationCacheData> {
+
+    @Override
+    public RegistrationCacheData loadFromSource(String key) {
+        return null;
```

Comment:
@Dominys  I know. And I catch this exception in get method of abstract class CacheService [link](https://github.com/AlexKazmirchuk/cources.server/blob/22a3cbce0762f75be35206c93c8854d41b4469bb/softgroup/basis/common-cache/src/main/java/com/softgroup/common/cache/impl/CacheService.java#L44-L52)

```java

public abstract class CacheService<T> implements Cacheable<T> {

...
// some other code
...

@Override
    public  T get(String key){
        try {
            return cache.get(key);
        } catch (ExecutionException e) {
            return null;
        } catch (CacheLoader.InvalidCacheLoadException ex){
            return null;
        }
    }
```